documentation/source/users/rmg/kinetics.rst

```diff
@@ -24,26 +24,36 @@ parameters
 +-------+------------------------------------------------------+
 |Rank   |Example methods                                       |
 +=======+======================================================+
-|Rank 1 |Experiment                                            |
+|Rank 1 |Experiment/FCI                                        |
 +-------+------------------------------------------------------+
-|Rank 2 |High level calculation: CCSD(T)-F12, MS-Tor, 2d rotors|
+|Rank 2 |W4/HEAT with very good (2-d if necessary) rotors      |
```

Comment:
Could we add a simple glossary or references to explain terms like  FCI / W4 / HEAT ?

Reply:
That would be a nice idea, I don't really feel qualified to do that myself though.

Reply:
@cgrambow, could you assist in writing a short explanation or adding links, so we know how to use the new ranks?

Reply:
Here are some bullet points about the acronyms. Let me know if you want this in a different format or in more detail.
- FCI (full configuration interaction): Exact solution to Schrödinger equation within the chosen basis set and Born-Oppenheimer approximation; possible for about 12 electrons with reasonably sized basis set (cost grows factorially with number of electrons)
- Wn (Weizmann-n): Composite methods often with sub-kJ/mol accuracies; W1 is possible for about 9 heavy atoms; W1 aims to reproduce CCSD(T)/CBS; W4 aims to reproduce CCSDTQ5/CBS; https://aip.scitation.org/doi/full/10.1063/1.2348881
- HEAT (high accuracy extrapolated ab initio thermochemistry): Also sub-kJ/mol accuracies; essentially CCSDTQ with various corrections; similar in cost to Wn I think; https://aip.scitation.org/doi/10.1063/1.1811608; https://aip.scitation.org/doi/full/10.1063/1.2835612
- CBS (complete basis set): Typically obtained by extrapolating to the complete basis set limit, i.e., successive cc-pVDZ, cc-pVTZ, cc-pVQZ, etc. calculations with some extrapolation formula
- CCSD(T)-F12: Coupled cluster with explicit electron correlation; chemical accuracy (1 kcal/mol) possible with double-zeta basis sets; https://aip.scitation.org/doi/full/10.1063/1.3054300

Some notes:
- CCSD(T)-F12/D2 should be CCSD(T)-F12/DZ, where DZ stands for double-zeta and refers to any double-zeta basis set, e.g., cc-pVDZ, def2-SVP, 6-31G*, ...
- Write B3LYP instead of B3lyp
- Maybe write "rotors if necessary" for the DFT methods?

Reply:
Added